src/iapetos/collector/ring.clj

```diff
@@ -120,14 +123,20 @@
   (->> (labels-for options request)
        (registry :http/exceptions-total)))
 
+(defn- safe [{:keys [exception-status]} f]
+  (if exception-status
+    (try (f) (catch Exception e e))
+    (f)))
+
 (defn- run-instrumented
   [{:keys [handler] :as options} request]
   (ex/with-exceptions (exception-counter-for options request)
     (let [start-time (System/nanoTime)
-          response   (handler request)
+          response   (safe options #(handler request))
```

Comment:
I don't think you need safe, I think it suffices to do 
```clojure
response (try (handler request) (catch Exception e e))
```

Reply:
we can do this but it introduces a subtle difference in behavior:
it means we will always catch an exception thrown in `(handler request)` and this exception will always be counted in the totals
it also means we will have to provide some default value to `:exception-status` since now there is no conditional relying on this value being provided

what do you think

Reply:
Let's go with your solution 👍

Reply:
If you could pass exception status as a param like

```clojure
(defn- maybe-safe [catch-exceptions? f]
  (if catch-exceptions?
     (try (f) (catch Exception e e))
     (f)))
```
That'd be great.

You could then call it with `(safe (:exception-status options) #(handler request))`

Reply:
sure e8e5715